test/jasmine/tests/gl3d_plot_interact_test.js

```diff
@@ -384,7 +384,7 @@ describe('Test gl3d plots', function() {
         .then(done);
     });
 
-    it('@gl should display correct hover labels (mesh3d case)', function(done) {
+    it('@noCI @gl should display correct hover labels (mesh3d case)', function(done) {
```

Comment:
Does this test still pass locally using

```
(plotly.js) $ ./tasks/noci_test.sh
```

?

Reply:
Unfortunately no. The mock is unexpectedly rotated after the click possibly related to `uirevision`.
But I need to figure that out.`

Reply:
Yes please,

```
(plotly.js) $ ./tasks/noci_test.sh
```

should (always :smile: ) pass.